pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "terraform-worker"
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
So, this is fine but maybe the team should chat about how we want to handle version numbers. this change doesn't seem to be something that warrants a minor release, no functionality changes, etc..... good for now but we should consider what we want our versions to mean.